packages/plugins/posthog-ui/src/lib/directives/posthog-track.directive.ts

```
@@ -0,0 +1,66 @@
+ * >
+ *   Submit
+ * </button>
+ */
+@Directive({
+	selector: '[phTrack]'
+})
+export class PostHogTrackDirective implements OnInit {
+	@Input('phTrack') eventName!: string;
+	@Input('phProperties') properties: Record<string, any> = {};
+	@Input('phOnInit') captureOnInit = false;
+	@Input('phEventType') eventType: string = 'click'; // default to click
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)